static/js/src/advantage/subscribe/renderers/form-renderer.js

```diff
@@ -184,11 +190,15 @@ function renderSummary(state) {
     buyButton.dataset.previousPurchaseId = previous_purchase_id;
   }
 
-  // Monthlty is only available for infra and essential
-  if (state.feature !== "infra" || state.support !== "essential") {
-    billingSection.classList.add("u-hide");
-  } else {
+  if (periods.length > 1) {
     billingSection.classList.remove("u-hide");
+    let options = "";
+    periods.forEach((period) => {
```

Comment:
Sorry, if I am not being clear but truly dynamic will also cause issues as we need middleware logic. What I meant by "add items to the select box in a clear known way" was to conditionally add known options if the period exists in the array.